src/component/Header/index.jsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react"
+
+function Header() {
+
+    const [unpaid, setUnpaid] = useState (0)
+
+    useEffect( () => {
+        fetch('https://invoicing-api.dev.io-academy.uk/invoices?status=2')
+        .then((res) => {
+            return res.json()
+        })
+        .then((unpaidInvoices) => {
+            setUnpaid(unpaidInvoices.data.length)
+        })
```

Comment:
The then blocks should be indented by 1 more level

Reply:
Also, you should be consistent with the InvoiceContainer and use short hand arrow funtions